### PR TITLE
Update README.md repo discontinued

### PR DESCRIPTION
**Pull request by dantekind**
https://peltarion.atlassian.net/browse/PLAT-10113

There is a new repo for Sidekick at https://github.int.midasplayer.com/ml-platform/peltarion-sidekick

This repo should be discontinued.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,222 +1,5 @@
...
-
-### Compatible filetypes
-The filetypes compatible with sidekick may shown by:
-```python
-print(sidekick.encode.FILE_EXTENSION_ENCODERS)
-```
-
-# Examples
-Examples of how to use sidekick are available at: [examples/](examples/).
+Kingsters please use https://github.int.midasplayer.com/ml-platform/peltarion-sidekick
```
Comment (psan-peltarion) [inline README.md:5]: https://github.int.midasplayer.com/ml-platform/peltarion-sidekick has not been created yet, or is it just me not able to access it?

**Review by psan-peltarion — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (dantekind) [reply README.md:5]: My bad, I accidentally made it private. Please try again

Comment (psan-peltarion) [reply README.md:5]: Looks good :+1:

Comment (dantekind) [reply README.md:5]: Only problem is that it's waiting for some Travis CI check to be reported, but there seems to be no such check running...?

**Pull request merged**